.github/pull_request_template.md

```diff
@@ -1,23 +1,30 @@
+# Pull Requests
+
+A Pull Request should close a **single** issue; multiple issues can be closed when the issues are **small and related**, but that should be an exception not the rule. Please
```

Comment:
Can this section/text be consolidated into the text under Related Issue?

Reply:
Good idea!